README.md

```diff
@@ -1,37 +1,48 @@
 # MyPortfolio
 #This is MyPortfolio Project
-    This is simple Project.Which has header, Section.Inside header there is logo and menu icon.
-    Inside section some pictures. My itroduction in paragraph.I add a work section where details of my work. And after my work is my skill section where I add detailas of languages, frameworks and skills.And at the bottom there is footer, and inside footer I added a contact form.
-    In the stylesheet add media quries to make the portfolio desktop responsive.
+   This is a mobile-first portfolio project built using Semantic HTML and CSS. It includes the navigation bar, the headline, a list of past projects(which includes skills, languages and frameworks used), the about section and a contact form.
+    Also add github Pages to make my Portfolio online available.
+  
 
   ![screenshot](./Images/newPort.png)
-
-  ![screenshot](./Images/s1.png)
-
-  ![screenshot](./Images/s2.png)
-
-  ![screenshot](./Images/s3.png)
-  ![screenshot](./Images/s4.png)
-  ![screenshot](./Images/s5.png)
-  ![screenshot](./Images/form.png)
   
   ## Built With
 
 - Html,CSS
 - Git, Github
 - Visual Studio Code
 
+## Live Demo
+
+[Live Demo Link](https://ridaarif98.github.io/NewPortfolio/)
+
 
 
 ## Getting Started
 
 **This is an example of how you may create your mobile responsive porfolio**
 
-##Prerequisites
+## Prerequisites
+
+### Local Usage
+- Create repository
+- Clone it local device using git
```

Comment:
[OPTIONAL] [Highly Recommended]

- You can also considering to add how to clone (`i.e. git clone https://github.com/ridaarif98/NewPortfolio.git`). Think about the person who don't know how to clone the project.